src/main/java/bio/terra/cli/command/config/List.java

```diff
@@ -0,0 +1,22 @@
+package bio.terra.cli.command.config;
+
+import bio.terra.cli.context.GlobalContext;
+import java.util.concurrent.Callable;
+import picocli.CommandLine;
+
+/** This class corresponds to the third-level "terra config list" command. */
+@CommandLine.Command(
+    name = "list",
+    description = "List all configuration properties and their values.")
+public class List implements Callable<Integer> {
+
+  @Override
+  public Integer call() {
+    GlobalContext globalContext = GlobalContext.readFromFile();
+
```

Comment:
I like the human readable description for the parameter; it might be useful to also print out the corresponding parameter name as well.

Reply:
Good idea. Added the parameter name to the `list` output.